frontend/src/store/helpers/profileHelper.ts

```diff
@@ -1,5 +1,24 @@
 import axiosInstance from "services/axios";
+import { ProfileInterface } from "interfaces";
 
 export const getProfilesOfUser = (userID: string) => {
     return axiosInstance.get(`/userProfiles/${userID}`).then((data) => data.data);
 };
+
+export const setActiveProfile = (profile: ProfileInterface) => {
+    localStorage.setItem("activeAccount", JSON.stringify(profile._id));
+    return profile;
+};
+
+export const setActiveProfileAuto = (profiles: ProfileInterface[]) => {
+    const localStorageProfile = localStorage.getItem("activeAccount") || "";
```

Comment:
suggestion: you could refactor this so that the `setActiveProfileAuto` calls `setActiveProfile` inside of it, since it does the same thing.

The only thing that the `setActiveProfileAuto` does differently is getting the profileId from localStorage, which you can pass to the other method.